### ISSUE TITLE
Any progress migrate from pgx to pgrx?

### ISSUE THREAD

**Issue by elvizlai** [supabase/pg_graphql#355]
# Summary
[summary]: #summary

Since pgx renamed as pgrx, any plan to migrate?

# Rationale
[rationale]: #rationale

Keep tracking pgrx lib

# Design
[design]: #design

An dense explanation in sufficient detail that someone familiar with the
project could implement the feature. Specifics and corner cases should be covered.

# Examples
[examples]: #examples

Illustrations and examples to clarify descriptions from previous sections.

# Drawbacks
[drawbacks]: #drawbacks

What are the negative trade-offs?

# Alternatives
[alternatives]: #alternatives

What other solutions have been considered?

# Unresolved Questions
[unresolved]: #unresolved-questions

What parts of problem space or proposed designs are unknown or TBD?

**Comment by olirice** [comment]
yes we do. I need to find an hour to merge #349, #325, and then do all the renaming 

#325 removes a dependency that isn't compatible with pgx after 0.7.1 due to changes to the SPI interface
